lib/src/domain/usecases/get_recipes_usecase.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:get/get.dart';
+
+import '../../core/usecases/i_usecase.dart';
+import '../../data/models/recipe_model.dart';
+import '../../data/repositories/recipes_repository.dart';
+
+class GetRecipesUseCase implements IUseCase<List<RecipeModel>, String> {
+  final RecipesRepository _recipesRepository = Get.find();
```

Comment:
Could you add the type of the find that you are using, in this case, you can do Get.find<RecipeRepository>.

Reply:
Done